lib/rory.rb

```diff
@@ -1,11 +1,7 @@
 ENV['RORY_ENV'] ||= ENV['RORY_STAGE'] || ENV['RACK_ENV'] || 'development'
 
 if ENV['RORY_STAGE']
-  puts "\n\tDEPRECATION: use 'RORY_ENV' instead of 'RORY_STAGE'\n\n"
-else
-  # Set RORY_STAGE to the default as well, since people might
-  # still be using it
-  ENV['RORY_STAGE'] = ENV['RORY_ENV']
+  puts "\n\tWARNING: use of 'RORY_STAGE' is ignored. Use 'RORY_STAGE' instead.\n\n"
```

Comment:
Wow, this is a confusing message. :)  I'm guessing you meant the latter "RORY_STAGE" to be "RORY_ENV"?